scripts/check-public-api.spec.ts

```diff
@@ -22,28 +23,13 @@ describe('check-public-api', () => {
         }
       }
     });
-    expect(indexFiles('root', 'index.ts')).toEqual([
+    expect(indexFiles('root', '**/index.ts')).toEqual([
```

Comment:
Also a second check to ensure that the content of the folder is parsed and comared to the content would be good.